pkg/operator/hive/sharded_controllers.go

```diff
@@ -74,89 +130,85 @@ func (r *ReconcileHiveConfig) deployClusterSync(hLog log.FieldLogger, h resource
 	}
 
 	if level := hiveconfig.Spec.LogLevel; level != "" {
-		clusterSyncContainer.Args = append(clusterSyncContainer.Args, "--log-level", level)
+		container.Args = append(container.Args, "--log-level", level)
 	}
 
-	if syncSetReapplyInterval := hiveconfig.Spec.SyncSetReapplyInterval; syncSetReapplyInterval != "" {
-		syncsetReapplyIntervalEnvVar := corev1.EnvVar{
-			Name:  constants.SyncSetReapplyIntervalEnvVar,
-			Value: syncSetReapplyInterval,
-		}
-
-		clusterSyncContainer.Env = append(clusterSyncContainer.Env, syncsetReapplyIntervalEnvVar)
+	if c.containerCustomization != nil {
+		c.containerCustomization(hiveconfig, container)
 	}
 
 	hiveNSName := GetHiveNamespace(hiveconfig)
 
 	// Load namespaced assets, decode them, set to our target namespace, and apply:
-	for _, assetPath := range namespacedAssets {
-		if err := util.ApplyAssetWithNSOverrideAndGC(h, assetPath, hiveNSName, hiveconfig); err != nil {
-			hLog.WithError(err).Error("error applying object with namespace override")
+	for _, a := range namespacedAssets {
+		// YOU ARE ALSO HERE: convert this func to take bytes, like you did for delete
```

Comment:
Intentional?

Reply:
Heh, thanks, a note-to-self as I left my desk for the night, forgot to scrub it. Will do in a fup.